src/main/java/com/pub/wysio/entity/Customer.java

```diff
@@ -0,0 +1,56 @@
+package com.pub.wysio.entity;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name="CUSTOMER")
+public class Customer {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name="NICKNAME")
+    private String nickname;
+    @Column(name="LIVERSTRENGTH")
+    private double liverStrength;
+    @Column(name="BICEPSIZE")
```

Comment:
I think columns get the name of the field by default, so these `name` parameters are unnecessary